starters/graphql-serverless-contentful/.jest/set-env-vars.ts

```diff
@@ -0,0 +1,6 @@
+import * as dotenv from 'dotenv';
+
+// load dotenv config
+const { parsed } = dotenv.config();
+
+process.env = { ...process.env, ...parsed };
```

Comment:
There's no newline at the end of this file. Do you have your editor/IDE configured to use the `.editorconfig` file?

Reply:
Also have a look at @ihardz comment here, that's a good change as I don't expect devs to have set up their environment or dot files for the tests to work.

https://github.com/thisdot/starter.dev/pull/835/files#r1082622089